transform/snowflake-dbt/models/mattermost/hourly/server_fact.sql

```diff
@@ -432,9 +427,9 @@ WITH sdd AS (
         , max(server_activity.guest_accounts) as guest_accounts
         , max(server_activity.incoming_webhooks) as incoming_webhooks
         , max(server_activity.outgoing_webhooks) as outgoing_webhooks
-        , max(server_activity.boards) as boards
-        , max(server_activity.boards_views) as boards_views
-        , max(server_activity.boards_cards) as boards_cards
+        , max(fba.boards) as boards
```

Comment:
are we trying to get current number of boards for a server or max number of boards at any given time?